<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Zayns-Air.home" ignored="0" name="Default test(failed)" tests="1" failures="1" timestamp="2023-03-07T07:37:16 EST" time="53.004" errors="0">
  <testcase name="verifyHomepageTitle" time="52.956" classname="com.edureka.testng.Example">
    <failure type="java.lang.AssertionError" message="expected [Instructor-Led Online Training with 24x7 Lifetime Support | Edureka] but found [Instructor-Led Online Training with 24X7 Lifetime Support | Edureka]">
      <![CDATA[java.lang.AssertionError: expected [Instructor-Led Online Training with 24x7 Lifetime Support | Edureka] but found [Instructor-Led Online Training with 24X7 Lifetime Support | Edureka]
at com.edureka.testng.Example.verifyHomepageTitle(Example.java:22)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyHomepageTitle -->
</testsuite> <!-- Default test(failed) -->
